Tests/Blast/xml_2226_tblastn_007_err.xml

```diff
@@ -0,0 +1,23 @@
+this is an empty file to test the method NCBIXML.handle()
```

Comment:
This comment does not make sense - do you mean this is a real XML file but with some unwanted text at the start?